target/linux/lantiq/patches-4.19/4053-NET-reactivate-ndo_do_ioctl-fallback.patch

```diff
@@ -0,0 +1,27 @@
+From c9701bab45cf3fa244033fcf5e4341a1de627807 Mon Sep 17 00:00:00 2001
+From: Quallenauge <Hamsi2k@freenet.de>
+Date: Mon, 21 May 2018 23:55:01 -0100
+Subject: [PATCH] VGV952CJW33-E-IR: Revert "wireless: wext: remove ndo_do_ioctl
+ fallback" This reverts commit 8bfb3676606454ffec836f56c5dc3e69dfc0956a.
+
+The ralink inic driver currently uses this for controlling the network
+adapters. In future versions the driver has to be modernized to comply
+with future kernel drivers.
+---
+ net/wireless/wext-core.c | 3 +++
+ 1 file changed, 3 insertions(+)
+
+Index: linux-4.14.41/net/wireless/wext-core.c
+===================================================================
+--- linux-4.14.41.orig/net/wireless/wext-core.c
++++ linux-4.14.41/net/wireless/wext-core.c
```

Comment:
Does not look like it should be in patches-4.19?

Reply:
It is the same patch in 4.14

Reply:
Okay, technically the 4.14 here is irrelevant. Nevertheless, it is misleading.
I'd prefer to have the four lines lines
```
Index: linux-4.14.41/net/wireless/wext-core.c
===================================================================
--- linux-4.14.41.orig/net/wireless/wext-core.c
+++ linux-4.14.41/net/wireless/wext-core.c
```
reduced to
```
--- a/net/wireless/wext-core.c
+++ b/net/wireless/wext-core.c
```

which will have the same effect, but be less awkward.

The optimal solution would obviously be to create real patches based on real kernel repo, but I won't annoy you by requesting that.

Reply:
i am updating wait a littlebit